styles/main.css

```diff
@@ -71,76 +82,120 @@ ul {
     padding-left: 2em;
     padding-right: 2em;
     margin-bottom: 3em;
-    border-left:1px solid rgba(0, 0, 0, 0.3);
-    border-right:1px solid rgba(0, 0, 0, 0.3);
-    border-bottom:1px solid rgba(0, 0, 0, 0.3);
+    border-left: 1px solid rgba(0, 0, 0, 0.3);
+    border-right: 1px solid rgba(0, 0, 0, 0.3);
+    border-bottom: 1px solid rgba(0, 0, 0, 0.3);
     box-shadow: 0 0 4px rgba(255, 0, 0, 0.3);
     background-color: #ffe4e1;
     -webkit-box-shadow: 4px 4px 4px rgba(255, 0, 0, 0.3);
     -moz-box-shadow: 4px 4px 4px rgba(255, 0, 0, 0.3);
 }
-.hype {list-style: none;
-              padding:0
-              }
-body{padding-top: 0;
-     padding-bottom: 0;
-     }
+
+.hype {
+    list-style: none;
+    padding: 0
+}
+
+body {
+    padding-top: 0;
+    padding-bottom: 0;
+}
+
 .mugshot img {
     box-shadow: 2px 2px 3px rgba(0, 0, 0, 0.5);
 }
+
 footer hr {
-    width:50vw;
-    min-width:300px;
-    margin-top:3em;
-    margin-bottom:2em;
+    width: 50vw;
+    min-width: 300px;
+    margin-top: 3em;
+    margin-bottom: 2em;
     border-top: 3px solid #00ffff;
-    border-bottom:8px dotted #00ffff;
+    border-bottom: 8px dotted #00ffff;
     border-top-left-radius: 8px;
     border-top-right-radius: 8px;
     border-left: none;
     border-right: none;
-}        
+}
+
 * {
-    box-sizing: border-box; }
-.paperclip {background-image: url("/images/paperclip.png");

```

Comment:
This path is relative to the root domain.  You need to use `../` instead to select the path relative to the current directory (styles) and then move up one from there (../) to find the images folder